### ISSUE TITLE
Question about the default setting of the target name

### ISSUE THREAD

**Issue by mayyadeng** [atomistic-machine-learning/schnetpack#549]
As mentioned in the QM9 tutorial, the target and output names are kept the same by default. When developing my own model, I also chose the default setting. 

However, I have encountered an issue with this default setting during the calculation of the loss. Since their names are the same, after each training step, the results in the dataloader get overwritten due to the same name. As a result, when calculating the loss, it is actually not computing the loss between the actual values and predicted values, but rather the loss between the (n-1)th epoch and the nth epoch.

It is possible that I have misunderstood something, and I am looking forward to your explanation.

**Comment by MetaEnigma97** [comment]
I have the same question about this. Do you have a better solution or understanding now?
